Airports/EGGP/VRPs.txt

```diff
@@ -1,21 +1,21 @@
 *Aintree:N053.28.36.000:W002.56.35.000
```

Comment:
This would interfere with the `Kirkby` VRP I think:
![EGGP Aintree](https://user-images.githubusercontent.com/61326713/226187459-16c96a34-89e4-47c3-bd5e-ce85b23b1867.png)